tests/test_install.py

```diff
@@ -751,7 +751,8 @@ def test_download(self, mock_urlopen: MagicMock):
         with envvar("AUTOBUILD_GITHUB_TOKEN", None):
             autobuild_tool_install.download_package("https://example.org/foo.tar.bz2")
             mock_urlopen.assert_called()
-            got_req = mock_urlopen.mock_calls[0].args[0]
+            _, args, _ = mock_urlopen.mock_calls[0]
+            got_req = args[0]
```

Comment:
Is this a Python-version-dependent change to the mock API? I don't understand why these changes are needed for backing off to support Python 3.7.

Also, couldn't this be more succinctly expressed as:

    got_req = mock_urlopen.mock_calls[0][1][0]

I guess that's less clear though.

Reply:
These tests were failing on 3.7 due to differences in the mocking API between 37 and later version. The destructuring was done for clarity and to resolve the incompatibility.

Reply:
> differences in the mocking API between 37 and later version

Thanks, that's what I wanted to know. Presumably later versions are also compatible with the 3.7 API.